model/export/AssemblyExporterService.php

```diff
@@ -150,9 +150,13 @@ protected function doExportCompiledDelivery($path, core_kernel_classes_Resource
             $data['dir'][$id] = $directory->getPrefix();
         }
 
-        $runtime = $compiledDelivery->getResource($this->getProperty(DeliveryAssemblyService::PROPERTY_DELIVERY_RUNTIME));
-        $serviceCall = tao_models_classes_service_ServiceCall::fromResource($runtime);
-        $data['runtime'] = base64_encode($serviceCall->serializeToString());
+        $runtime = $compiledDelivery->getOnePropertyValue(
+            $this->getProperty(DeliveryAssemblyService::PROPERTY_DELIVERY_RUNTIME)
+        );
+        $serviceCall = $runtime instanceof core_kernel_classes_Resource
+            ? tao_models_classes_service_ServiceCall::fromResource($runtime)
+            : tao_models_classes_service_ServiceCall::fromString((string)$runtime);
+        $data['runtime'] = base64_encode(json_encode($serviceCall));
```

Comment:
Service call already have a `jsonSerialize` [method](https://github.com/oat-sa/tao-core/blob/master/models/classes/service/class.ServiceCall.php#L211).
Maybe it's better to use it?

Reply:
`json_encode()` invokes that method.
That's the whole point of `tao_models_classes_service_ServiceCall` implementing `JsonSerializable` ;)

Reply:
Ah, sht, didn't see implementation, sorry :facepalm: 
Definitely need to sleep more...